docs/usage/cards/._example.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Example Card Page
+---
+
+## __card_name_1__
+
+Card description should be here.
+
+|Variable|Default|Required|Notes|
+|----------|----------|------|------|
+|name| | X | |
+|background| `true` | X | Enable background |
+
+![example-image](img/example.png)
```

Comment:
I don't know if I like the idea of each subdirectory containing its own assets. I'm more in favour of putting all the assets in a central folder in the root directory of the docs. Any other opinions on this?

Reply:
It can also be put under assets. But could make it harder to get the correct file. From my experience this works pretty well.

Reply:
If you look at cms like wordpress or drupal, etc., they don't have an asset hanging on every page but have a central place for assets that are referenced from the respective page.
There are maybe situations where the same asset is used on multiple pages. Then you would have to either copy it unnecessarily or create strange cross-references.

Reply:
That is true indeed. So should we go for assets/img/..